drift/lib/src/web/sql_js.dart

```diff
@@ -70,7 +70,7 @@ class _QueryExecResult {
 class _SqlJsStatement {
   external void bind(List<Object?> values);
   external bool step();
-  external List<Object?> get();
+  external List<Object?> get(Object? params, dynamic config);
```

Comment:
`jsify` has mediocre codegen and performance, we can write this like this:

```dart
@JS
@anonymous
class _SqlJsStatementGetOptions {
  external _SqlJsStatementGetOptions({bool useBigInt = false});
}

// And then use
external List<Object?> get(Object? params, _SqlJsStatementGetOptions config);
get(null, _SqlJsStatementGetOptions(useBigInt: useBigInt))
```

dart2js will emit a direct JS object literal when creating anonymous JS classes which is what we want.

Reply:
Thanks. I thought it is not the best solution, but the docs for jsinterop is almost zero.